README.md

```diff
@@ -58,7 +58,11 @@ Additional fields are NOT allowed for any object type.
         "type": "Package",
         "id": "1",
         "name": "UFO-S",
-        "description": "A commitment-based ontology of services.",
+        "description": {
+            "eng": "A commitment-based ontology of services.",
+            "spa": "Una ontología de servicios basada en el concepto de compromiso.",
+            "por": "Uma ontologia dos serviços baseada no conceito de compromisso.",
```

Comment:
I propose we use IANA language tags (as it is done in RDF, XML, HTML...).

See https://www.w3.org/International/articles/language-tags/ and https://www.iana.org/assignments/language-subtag-registry/language-subtag-registry

Here as some examples: 

- Portuguese is `pt`
- English is `en`
- Italian is `it`

The nice feature of IANA language tags is that we can choose to represent language variants:

- Brazilian Portuguese is `pt-BR`
- American English is `en-US`
- Canadian French is `fr-CA`
- Chinese written with Simplified script is `zh-Hans`